test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secret2 *api.Secret) {
```

Comment:
So you are saying keep it as 'secret2' in the `It() block` and change `doSecretE2EWithoutMapping()` to take a string for name ?